File: FaktoriyelDongu.py
#Python3 için yazılmıştır.
def faktoriyel(sayi):
    sonuc=1
    while sayi>=1:
        sonuc=sonuc*sayi
        sayi-=1
    return sonuc

deger=int(input("Bir sayi giriniz: "))
print("Faktoriyeli: {}".format(faktoriyel(deger)))

File: FaktoriyelDongu.c
#include<stdio.h>

int main(){
   
    int sayi,faktoriyel=1;
    
    printf("Bir sayi giriniz: ");
    scanf("%d",&sayi);
    
    while (sayi>0) {
        faktoriyel*=sayi;
        sayi--;
    }
    
    printf("Faktoriyeli: %d",faktoriyel);
    
    
    
    return 0;
}
